frontend/packages/console-dynamic-plugin-sdk/src/api/api.ts

```diff
@@ -35,6 +37,12 @@ export const consoleFetchJSON: ConsoleFetchJSON = require('@console/dynamic-plug
   .consoleFetchJSON;
 export const consoleFetchText: ConsoleFetchText = require('@console/dynamic-plugin-sdk/src/utils/fetch')
   .consoleFetchText;
+
+export const useActivePerspective: UseActivePerspective = require('@console/dynamic-plugin-sdk/src/perspective/useActivePerspective')
+  .default;
+export const PerspectiveDetector: React.FC<PerspectiveDetectorProps> = require('@console/dynamic-plugin-sdk/src/perspective/PerspectiveDetector')
+  .default;
```

Comment:
@vojtechszocs I added them here instead of a direct export which seem to not like my use of jsx.

Reply:
I don't see `PerspectiveDetector` as necessary in the SDK. This is more of an app scaffolding component.

Reply:
Good point, reverted location.